testsrc/main/mondrian/olap/HierarchyBugTest.java

```diff
@@ -72,6 +77,226 @@ public void testNoHierarchy() {
             fail(failStr);
         }
     }
+
+    /**
+     * Test case for <a href="http://jira.pentaho.com/browse/MONDRIAN-1126">
+     * MONDRIAN-1126:
+     * member getHierarchy vs. level.getHierarchy differences in Time Dimension
+     * </a>
+     */
+    public void testNamesIdentitySsasCompatible() {
+        propSaver.set(
```

Comment:
@YuryBY Running this test by itself will fail.  The [Time.Weekly] reference you have below is not resolvable using SSAS compatible naming (which would use [Time].[Weekly].

I'm guessing this succeeds when running the full test class because there's a sequence issue with the property value not applied.  Possibly the test is loading an already cached schema, which had the default properties set from an earlier test execution.